### PR TITLE
VZ-8842:PR to uptake new fluentd images to update deps

### PR DESCRIPTION
**Pull request by neeraj-n-singh**

### CODE REVIEW

File: v1.14/oraclelinux/Dockerfile [hunk 1]
```diff
@@ -1,15 +1,18 @@
-FROM container-registry.oracle.com/os/oraclelinux:7-slim
+FROM container-registry.oracle.com/os/oraclelinux:8-slim
 LABEL maintainer ="Verrazzano developers <verrazzano_ww@oracle.com>"
-LABEL Description="Fluentd docker image for Oracle Linux 7" Vendor="Oracle" Version="1.12.3"
+LABEL Description="Fluentd docker image for Oracle Linux 8" Vendor="Oracle" Version="1.12.3"
```
Comment (adiforluls) [inline v1.14/oraclelinux/Dockerfile]: Version should be `1.14.5`?

File: v1.14/oraclelinux/Dockerfile [hunk 2]
```diff
@@ -1,15 +1,18 @@
-FROM container-registry.oracle.com/os/oraclelinux:7-slim
+FROM container-registry.oracle.com/os/oraclelinux:8-slim
```
Comment (adiforluls) [inline v1.14/oraclelinux/Dockerfile]: Should this image be coming from ghcr? `ghcr.io/oracle/oraclelinux:8-slim`

Comment (neeraj-n-singh) [reply v1.14/oraclelinux/Dockerfile]: We are generally pulling from oracle, I can toggle to ghcr

**Review by adiforluls — APPROVED** [review]
LGTM

**Pull request merged**